src/app/app.route.ts

```diff
@@ -16,7 +17,8 @@ const appRoutes: Routes = [
         canActivate: [AuthGuard],
     },
     { path: 'landing', component: LandingComponent },
-    { path: 'dashboard-placeholder', component: DashboardComponent },
+    { path: 'user', component: UserComponent },
```

Comment:
I think you want to place the authGuard and adminGuard on both the user and admin route respectively